packages/react-accordion/Spec.md

```diff
@@ -0,0 +1,369 @@
+
+## Comparison of `@fluentui/react` and `@fluentui/react-northstar`
+
+- All mentions of v7 or v8 == `@fluentui/react` ([docsite](https://developer.microsoft.com/en-us/fluentui#/))
+- All mentions of v0 == `@fluentui/react-northstar` ([docsite](https://fluentsite.z22.web.core.windows.net/))
+
+There's no comparison to be done between the two libraries, since v8 hasn't implemented this component.
+
+## Issues
+
+- [#16553](https://github.com/microsoft/fluentui/issues/16553)
+- [#16452](https://github.com/microsoft/fluentui/issues/16452)
+- [#12953](https://github.com/microsoft/fluentui/issues/12953)
+- [#12238](https://github.com/microsoft/fluentui/issues/12238)
+
+## API
+
+The `Accordion` should implement a `children` based API as is the standard across all the surveyed alternatives as a part of Open UI research in [Prior Art](#prior-art). The component will leverage the use of `context` in the interaction and data flows of child components.
+
+Sample usages will be given in the following section of this document [Sample code](#sample-code)
+
+### Accordion
+
+The root level component serves context and common API between all children.
+
+| Prop name          | Type                | Default Value | Details                                                                                                                    |
+| ------------------ | ------------------- | ------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| multiple           | boolean             | false         | Allows multiple panels to be expanded                                                                                      |
+| collapsible        | boolean             | false         | Allows multiple panels to be collapsed                                                                                     |
+| expandIconPosition | "start" or "end"    | "start"       | Position of the icon to indicate expansion                                                                                 |
+| onToggle           | ToggleEventListener |               | Equivalent to onToggle on AccordionPanel element                                                                           |
+| open               | number or number[]  | undefined     | The index (or a list of indexes) of opened items. If this property is being used than the element is considered Controlled |
+| defaultOpen        | number or number[]  | undefined     | The index (or a list of indexes) of opened items by default.                                                               |
+
+```tsx
+interface ToggleEventListener {
+  (index: number): void;
+}
```

Comment:
FYI: there is an unfinished RFC about event handlers, #12588. But, anyway I suggest to include `event` as first param, for now.